portfolio/src/main/webapp/index.html

```diff
@@ -15,5 +15,11 @@
         <p id="date"></p>
       </ul>
     <h1>Hello! I am Carolyn.</h1>
+    <form action="/data" method="POST">
+      <p>Enter your comment here</p>
+      <textarea name="text-input"></textarea>
+      <br/><br/>
```

Comment:
No need to close p or br tags: https://google.github.io/styleguide/htmlcssguide.html#Document_Type
Any reason why you are using 2 brs here rather than a p?

Reply:
I fixed this too.